### ISSUE TITLE
Null safety

### ISSUE THREAD

**Issue by peterscodee** [corsac-dart/jwt#22]
This is a request to add support for null safety to this package. See the [migration guide](https://dart.dev/null-safety/migration-guide) for details about enabling null safety.

**Comment by technolion** [comment]
With the release of flutter beta 2.2.0 today, this issue git more pressing, because it requires `mockito` ^5.0.0 which in turn requires `crypto` ^3.0.0.

```
Because mockito >=5.0.0 depends on analyzer ^1.0.0 which depends on crypto ^3.0.0, mockito >=5.0.0 requires crypto ^3.0.0.
Because analyzer ^1.0.0 depends on crypto ^3.0.0 and corsac_jwt 0.4.0 depends on crypto ^2.1.5, analyzer ^1.0.0 is incompatible with corsac_jwt 0.4.0.
```

**Comment by MisterJimson** [comment]
Anyone know of a solid alternative package?

**Comment by pulyaevskiy** [comment]
Fixed by #21
